components/raftstore/src/store/fsm/peer.rs

```diff
@@ -5724,7 +5724,9 @@ where
         // When Lightning or BR is importing data to TiKV, their ingest-request may fail
         // because of region-epoch not matched. So we hope TiKV do not check region size
         // and split region during importing.
-        if self.ctx.importer.get_mode() == SwitchMode::Import {
+        if self.ctx.importer.get_mode() == SwitchMode::Import
+            || self.ctx.importer.regional_import_mode()
```

Comment:
We should use region_in_import_mode instead.

Reply:
The reason why I use this is the same as the reply above.